ui/app/routes/vault/cluster/secrets/backend/list.js

```diff
@@ -80,7 +80,7 @@ export default Route.extend({
       ssh: 'role-ssh',
       transform: this.modelTypeForTransform(tab),
       aws: 'role-aws',
-      pki: tab === 'certs' ? 'pki-certificate' : 'role-pki',
+      pki: tab === 'certs' ? 'pki/pki-certificate' : 'pki/pki-role',
```

Comment:
changed name from role-pki to pki-role to match the naming conventions of the PKI engine.